### ISSUE TITLE
18Texas - Change Teleport Function

### ISSUE THREAD

**Issue by scottredracecar** [tobymao/18xx#9232]
After some playtesting feedback, I would like to change how the teleport operates in this game. This will be the final playtesting change for 18Texas.

My original intent was to have the teleport function like 18Chesapeake/1830 teleport where the token is placed only after the tile is laid by that company. However, after more playtesting feedback, I think a change is warranted.

The function/text should be revised as follows.

P2 Galveston and Red River Railway Company
Teleport to Houston. Phases 3-4. The owning president may upgrade to a #511 tile (at no cost) and place a token in hex I14 as the respective operation steps of a corporation for which they are president. The corporation does not need to have a route to this hex.

P5 New Orleans Pacific Railroad
Teleport to San Antonio. Phases 3-4. The owning president may upgrade to a #511 tile and place a token in hex J5 as the respective operation steps of a corporation for which they are president. The corporation does not need to have a route to this hex and the hex may be unbuilt or have a yellow tile. This private company comes with a 10% share of [XXXX].

**Comment by philcampeau** [comment]
Hey Scott, I know we had discussed this a while back, but the conversation is lost behind the slack archive paywall.

Can you reiterate the exact intentions you had? If I'm not mistaken, the goal was that the tile lay and the token were two separate abilities, that could be used separately, correct?

**Comment by scottredracecar** [comment]
Yes, separate steps. Also, I put in the errata doc that P2 should pay the normal cost (not free), so we should change that too.

**Comment by philcampeau** [comment]
And would the privates close after the abilities are exhausted?

**Comment by scottredracecar** [comment]
No, these don't close.
